ArticleTemplates/assets/js/modules/ads.js

```diff
@@ -165,6 +159,37 @@ define([
 
                 window.applyNativeFunctionCall("getMpuPosCallback");
 
+            },
+            // iOS Timer
+            iosTimer : function(time, yPos) {
+                setTimeout(function() {
+                    modules.runIosPoller(time, yPos);
+                }, 1000);
+            },
+            // iOS Count
+            runIosPoller : function(start, yPos) {
+                var thisNumber = parseInt(start, 10) + 1;
+                var yPolled;
+                if (thisNumber <= 1000) { 
```

Comment:
By doing this you're keeping the time running almost forever (1000 seconds) compared to the 20 seconds of the Android version.